packages/css-module/css-module.mjs

```diff
@@ -6,22 +6,30 @@ import postcss from 'postcss';
 import { stripExtras } from '@nodejs-loaders/parse-filename';
 
 
+/**
+ * @type {import('node:module').ResolveHook}
+ */
 async function resolveCSSModule(specifier, ctx, nextResolve) {
   const nextResult = await nextResolve(specifier);
 
   if (!stripExtras(specifier).endsWith('.module.css')) return nextResult;
 
   return {
     ...ctx,
+    // @ts-ignore
```

Comment:
This (and the other ones) can be removed after https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71493 is released